### PR TITLE
Add the GMT postdoc add

### PR DESCRIPTION
**Pull request by seisman**
Preview:
image

Is this what you want? Two links. One for the Google docs and one for the application link?

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -21,6 +21,12 @@
                <h2>News</h2>
 
                <ul>
+               <li>
+                  2020-06-23:
+                  <b>GMT postdoctoral position</b> available at SOEST, U of Hawaii at Mānoa.
+                  Click to <a href="LINK TO GOOGLE DOCS HERE"> view ad details</a> or
```
Comment (seisman) [inline index.rst:27]: Need to add the link to google docs here.

Comment (PaulWessel) [reply index.rst:27]: https://docs.google.com/document/d/15-ksiUotquaNYVnnpy8NKIWNU8pGPduf8__jZLSvGQE/edit?usp=sharing

Comment (seisman) [inline index.rst:27]:
```suggestion
                  Click to <a href="https://docs.google.com/document/d/15-ksiUotquaNYVnnpy8NKIWNU8pGPduf8__jZLSvGQE/edit?usp=sharing"> view ad details</a> or
```

Comment (seisman) [reply index.rst]: Done, but not sure it the permission is correct, since I can still edit it.

Comment (PaulWessel) [reply index.rst]: Sorry, this is apparently the direct link to the job ad
https://hcmweb.rcuh.com/psp/hcmprd_exapp/EMPLOYEE/HRMS/c/HRS_HRAM.HRS_APP_SCHJOB.GBL?Page=HRS_APP_JBPST&Action=U&FOCUS=Applicant&SiteId=3&JobOpeningId=220288&PostingSeq=1
The google docs has this link now.

File: index.rst [hunk 2]
```diff
@@ -21,6 +21,14 @@
                <h2>News</h2>
 
                <ul>
+               <li>
+                  2020-06-23:
+                  <b>GMT postdoctoral position</b> available at SOEST, U of Hawaii at Mānoa.
+                  Click to
+                  <a target="_blank" href="https://docs.google.com/document/d/15-ksiUotquaNYVnnpy8NKIWNU8pGPduf8__jZLSvGQE/edit?usp=sharing"> view ad details</a>
+                  or
+                  <a target="_blank" href="https://hcmweb.rcuh.com/psc/hcmprd_exapp/EMPLOYEE/HRMS/c/HRS_HRAM.HRS_APP_SCHJOB.GBL?FOCUS=Applicant&">apply online now</a>!
```
Comment (seisman) [inline index.rst:30]:
```suggestion
                  <a target="_blank" href="https://hcmweb.rcuh.com/psp/hcmprd_exapp/EMPLOYEE/HRMS/c/HRS_HRAM.HRS_APP_SCHJOB.GBL?Page=HRS_APP_JBPST&Action=U&FOCUS=Applicant&SiteId=3&JobOpeningId=220288&PostingSeq=1">apply online now</a>!
```

**Pull request merged**